dashboard/app/views/layouts/_header.html.haml

```diff
@@ -156,3 +156,9 @@
 - elsif level
   :javascript
     dashboard.header.buildProjectInfoOnly()
+
+-# If the page is cached, the sign-in button / user menu must be loaded separately
+-# by the client.
+- if @public_caching
+  :javascript
+    dashboard.header.buildUserMenu()
```

Comment:
I forget, is this also called in pegasus? For pegasus cached pages is the pairing status right?

Reply:
No, this is dashboard only.  The pairing status for pegasus may still be wrong.  (The main goal of this fix was to fix this issue for the levels and script overview pages, but it does fix it for all of dashboard.)